securityconfig/roles.yml

```diff
@@ -61,3 +61,35 @@ anomaly_full_access:
         - 'indices_monitor'
         - 'indices:admin/aliases/get'
         - 'indices:admin/mappings/get'
+
+# Allows users to read and download Reports
+reports_instances_read_access:
+  reserved: true
+  cluster_permissions:
+    - 'cluster:admin/opendistro/reports/instance/get_all'
+    - 'cluster:admin/opendistro/reports/instance/get'
+    - 'cluster:admin/opendistro/reports/menu/download'
+
+# Allows users to read and download Reports and Report-definitions
+reports_read_access:
+  reserved: true
+  cluster_permissions:
+    - 'cluster:admin/opendistro/reports/definition/get'
+    - 'cluster:admin/opendistro/reports/definition/get_all'
+    - 'cluster:admin/opendistro/reports/instance/get_all'
+    - 'cluster:admin/opendistro/reports/instance/get'
```

Comment:
What the difference between `get` and `get_all` ? Why we have to two API?

Reply:
yes they are 2 different APIs. thinking again may be it makes sense to rename get_all to list